src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,30 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import classNames from 'classnames';
+import './TodoInfo.scss';
+
+import { Todo } from '../../types/Todo';
+
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article className={classNames('TodoInfo', {
+      'TodoInfo--completed': completed,
+    })}
+    >
```

Comment:
let's format it better, if you have 2 function params and one of them spreads to a few lines of code then you should put each of them on a new line.
```suggestion
      className={classNames(
        'TodoInfo', 
        {
          'TodoInfo--completed': completed 
        }
      )}
```